web/styles/message_header.css

```diff
@@ -6,13 +6,13 @@
     text-overflow: ellipsis;
     white-space: nowrap;
     font-weight: 600;
-    font-size: calc(15em / 14);
```

Comment:
I'm kind of confused why this commit is in with the navbar stuff. Seems like it should be its own PR, and probably ought to address other icons and elements that are children of `.message_header`

Reply:
I'd definitely prefer a separate PR with screenshots. Message headers are a very high importance surface area, and we want to be very careful to avoid breaking anything involving them.

Reply:
33185

Reply:
Merged! I just pushed this PR with out this commit.